docs/contributing/contributing_package.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Contributing a Package
+---
+import Prerequisites from './components/prerequisites.jsx'
+
+This guide demonstrates how to bundle your own custom Expectations, Metrics, and Profilers into an official Great Expectations contributor package.
+
+<Prerequisites>
+
+* Created an account on [PyPi](https://pypi.org/account/register/)
+
+</Prerequisites>
+
+## Steps
+
+### 1. Install the `great_expectations_contrib` CLI Tool
+
+To streamline the process of contributing a package to Great Expectations, we've developed a CLI tool to
+abstract away some of the complexity and help you adhere to our codebases' best practices. Please 
+utilize the tool during your development to ensure that your package meets all of the necessary requirements.
+
+To install the tool, use the following commands:
+```bash
+# Ensure you are in the root of the `great_expectations/` codebase
+cd contrib/cli
+pip install -e great_expectations_contrib
+```
```

Comment:
I think we want to separate commands as much as possible, rather than having the user execute multiple commands at once.  I've also been avoiding having comment lines out of blocks that are likely to be copied directly into a terminal.

```suggestion
To install the tool, first ensure that you are in the root of the `great_expectations/` codebase:
```bash
cd contrib/cli
```

Next, use pip to install the CLI tool:
```bash
pip install -e great_expectations_contrib
```
```